### PR TITLE
CVE-2011-1112 and CVE-2014-7941

### PR DESCRIPTION
**Pull request by Beetee14**
Researched CVE-2011-1112 and CVE-2014-7941

### CODE REVIEW

File: cves/CVE-2014-7941.yml [hunk 1]
```diff
@@ -17,22 +17,22 @@ description_instructions: |
   keep too.
 description:
```
Comment (xdmds) [inline cves/CVE-2014-7941.yml]: please add a description

File: cves/CVE-2011-1112.yml [hunk 2]
```diff
@@ -15,23 +15,31 @@ description_instructions: |
...
+repo: https://skia.googlesource.com/skia
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-- :commit: 20106b615c3d11637864fcd4dd4de3356c858f2c
+- :commit: 116b2bcd2c4c4d4670c7b7e2ea597414713c37fb
   :note: ''
-vccs: []
+vccs: [ba09de4c4be66cc07790f23b0f3a925f47340e3e]
```
Comment (jonjxd) [inline cves/CVE-2011-1112.yml]: This is just a minor thing, but you don't need [] around your hash

File: cves/CVE-2011-1112.yml [hunk 3]
```diff
@@ -15,23 +15,31 @@ description_instructions: |
...
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
```
Comment (jonjxd) [inline cves/CVE-2011-1112.yml]: Interesting, I'll give this 5 upvotes

File: cves/CVE-2011-1112.yml [hunk 4]
```diff
@@ -15,23 +15,31 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
```
Comment (jonjxd) [inline cves/CVE-2011-1112.yml]: This refers to the first few lines, but you need a CWE number for this vulnerability

File: cves/CVE-2014-7941.yml (hunk 1 above)
Comment (Beetee14) [reply cves/CVE-2014-7941.yml]: Done

File: cves/CVE-2011-1112.yml (hunk 2 above)
Comment (Beetee14) [reply cves/CVE-2011-1112.yml]: I needed them for the CI to pass

File: cves/CVE-2011-1112.yml [hunk 5]
```diff
@@ -15,31 +15,39 @@ description_instructions: |
...
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  
+  When displaying a type of graphic known as an SVG, an invalid value could
```
Comment (HEATH3N) [inline cves/CVE-2011-1112.yml]: Is there any way you could be more specific than "invalid value" or was that information not easily discernable?

File: cves/CVE-2011-1112.yml (hunk 3 above)
Comment (HEATH3N) [reply cves/CVE-2011-1112.yml]: I'll give this 4 upvotes; I would want to know more of the specifics however, if possible.

**Pull request merged**